Packs/AWS/Integrations/AWS/AWS.py

```diff
@@ -1057,21 +1055,18 @@ def delete_security_group_command(client: BotoClient, args: Dict[str, Any]) -> C
         else:
             kwargs["GroupName"] = group_name
 
-        try:
-            remove_nulls_from_dictionary(kwargs)
-            response = client.delete_security_group(**kwargs)
-            if response.get("GroupId"):
-                return CommandResults(
-                    readable_output=f"Successfully deleted security group: {response.get('GroupId')}",
-                    raw_response=response,
-                )
-            else:
-                # If group_id was not found or no GroupId in response, raise an exception
-                AWSErrorHandler.handle_response_error(response)
-        except ClientError as err:
-            AWSErrorHandler.handle_client_error(err)
-
-        return CommandResults(readable_output="")
+        remove_nulls_from_dictionary(kwargs)
+        response = client.delete_security_group(**kwargs)
+        if response.get("ResponseMetadata", {}).get("HTTPStatusCode") == HTTPStatus.OK and response.get("GroupId"):
+            return CommandResults(
+                readable_output=f"Successfully deleted security group: {response.get('GroupId')}",
+                raw_response=response,
+            )
+        else:
+            # If group_id was not found or no GroupId in response, raise an exception
+            AWSErrorHandler.handle_response_error(response)
+        # Should Not Reach Here: handle_response_error should raise an error
+        return CommandResults(readable_output="Failed to Delete Security Group.")
```

Comment:
If should not get here, why do we have this line?

Reply:
<img width="1346" height="377" alt="image" src="https://github.com/user-attachments/assets/7bab973d-79f5-41a7-9df8-2c703bc49819" />
Pylance error.